hayeon/baekjoon/입출력연습/1085-직사각형에서탈출.js

```diff
@@ -0,0 +1,24 @@
+const readline = require("readline");
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const input = [];
+
+rl.on("line", function (line) {
+  input.push(line);
+}).on("close", function () {
+  // 풀이
+  const inputData = input[0].split(" ").map(Number);
+
+  const x = inputData[0];
+  const y = inputData[1];
+  const w = inputData[2];
+  const h = inputData[3];
+
+  const minData = [x, w - x, y, h - y];
+  const result = Math.min(...minData);
+
+  console.log(result);
```

Comment:
```suggestion
  const [x, y, w, h] = input[0].split(" ").map(Number);
  const result = Math.min(x, w - x, y, h - y);

  console.log(result);
```
- 풀이 잘봤습니다! 🙇‍♂️
- 구조 분해 할당을 이용하면  더 깔끔하게 입력값을 가져올 수 있습니다

Reply:
훨씬 간단해지는군요 감사합니다!👍🏻